frontendGraphImpl/src/main/java/com/virtuslab/gitmachete/frontend/graph/impl/coloring/GraphItemColorToJBColorMapper.java

```diff
@@ -1,9 +1,11 @@
-package com.virtuslab.gitmachete.frontend.graph.coloring;
+package com.virtuslab.gitmachete.frontend.graph.impl.coloring;
```

Comment:
Up to you, but I guess this mappers like this one can go into API subproject as long as they don't depend on anything else from `frontendGraphImpl`